### PR TITLE
Karma point issue

### PR DESCRIPTION
**Pull request by Afzal84**
Karma point issue (mobile responsive)
code was missing

### CODE REVIEW

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+      <!-- <a class="flex-4 mobile-margin show-all" [routerLink]="['/public/faq']" fragment="KarmaPoints"> -->
         <!-- <img class="info-icon" alt="info" src="/assets/icons/info.svg"> -->
-        <mat-icon class="cursor-pointer ws-mat-black60-text" [matTooltip]="translateLabels('tooltip', 'profileKarmapoints', '')">info_outline</mat-icon>
-      </a>
+        <!-- <mat-icon class="cursor-pointer ws-mat-black60-text" [matTooltip]="translateLabels('tooltip', 'profileKarmapoints', '')">info_outline</mat-icon>
+      </a> -->
+
+      <div class="flex flex-row flex-1 justify-between mobile-margin items-center">
+        <div class="flex">
+          <span class="mat-subheading-1 flex-4 mobile-margin" role="link" tabindex="0">
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html]: Please check for tabindex =0

File: library/ws-widget/collection/src/lib/profile-v2/profile-karmapoints/profile-karmapoints.component.html [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+          </span>
+          </div>
+          <div class="flex">
+          <a class="flex-4 show-all" (click)="navigateTo()" *ngIf="widgetData?.data?.length">
+            {{ 'profileKarmapoints.showAll' | translate}} &nbsp;&nbsp; >
+          </a>
+        </div>
+      </div>
+        <div class="flex mobile-margin">
+          <span class="flex-4">Karma Points history is available for last 3 months only.</span>
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/profile-v2/profile-karmapoints/profile-karmapoints.component.html]: hard coded text should be multilingual

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+          </div>
+          <div class="flex">
+            <a class="flex-4 mobile-margin show-all" [routerLink]="['/public/faq']" fragment="KarmaPoints">
+              <!-- <img class="info-icon" alt="info" src="/assets/icons/info.svg"> -->
+              <mat-icon class="cursor-pointer ws-mat-black60-text" [matTooltip]="translateLabels('tooltip', 'profileKarmapoints', '')">info_outline</mat-icon>
+            </a>
+        </div>
+      </div>
+        <div class="flex mobile-margin">
+          <span class="flex-4">Karma Points history is available for last 3 months only.</span>
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html]: it should be multilingual

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html (hunk 1 above)
Comment (Afzal84) [reply project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html]: removed, Checked locally, working fine

File: library/ws-widget/collection/src/lib/profile-v2/profile-karmapoints/profile-karmapoints.component.html (hunk 2 above)
Comment (Afzal84) [reply library/ws-widget/collection/src/lib/profile-v2/profile-karmapoints/profile-karmapoints.component.html]: Resolved

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html (hunk 3 above)
Comment (Afzal84) [reply project/ws/app/src/lib/routes/profile-v2/routes/profile-karmapoints/profile-karmapoints.component.html]: resolved

**Pull request merged**